modules/ROOT/edits/install-self-managed.adoc

```diff
@@ -0,0 +1,259 @@
+= Install Runtime Fabric on Self-Managed Kubernetes
+
+[IMPORTANT]
+====
+Installing Runtime Fabric on Self-Managed Kubernetes requires tasks that must be performed by both IT administrators and MuleSoft organization administrators.
+====
+
+Anypoint Runtime Fabric can be installed and operated on a AWS Elastic Kubernetes Service (EKS) or Azure Kubernetes Service (AKS) that you manage.
+
+
+== Before you Begin
+
+Before installing Anypoint Runtime Fabric in a self-managed Kubernetes environment, ensure the following:
+
+* You have read and understand the architecture and requirements outlined in xref:index-self-managed.adoc[Runtime Fabric on Self-Managed Kubernetes]
+* You have installed and configured your Kubernetes environment as follows:
++
+- Running an EKS or AKS Kubernetes environment. Other Kubernetes environments are not supported.
+- Running Kubernetes version 1.16.x to 1.18.x.
+- Running an ingress controller to send external requests to applications.
+
+== Step 1: Configure your Network to Support Runtime Fabric on Self-Managed Kubernetes
+
+[IMPORTANT]
+====
+Network configuration must be performed by an IT administrator.
+====
+
+Before installing or using Runtime Fabric on Self-Managed Kubernetes, ensure that the following ports and hostnames are configured correctly.
+
+=== Port Configuration
+
+To install or run Runtime Fabric, ensure that you have configured the following ports on your Kubernetes installation:
+
+[%header,cols="6*a"]
+.Port Configuration for Self-Managed Kubernetes
+|===
+| Port | Layer 4 Protocol | Layer 5 Protocol | Source | Destination | Description
+| 443 | TCP | HTTPS | Internet | All nodes | Allow inbound requests to Mule runtime servers
+| 443 | TCP | AMQP over WebSockets | All nodes | Internet | Anypoint Platform management services
+| 443 | TCP | HTTPS | All nodes | Internet | API Manager policy updates, API Analytics Ingestion, and Resource retrieval (application files, container images).
+| 5044 | TCP | Lumberjack | All nodes | Internet | Anypoint Monitoring, Anypoint Visualizer
+|===
+
+=== Hostname Configuration
+
+To function correctly, Runtime Fabric on Self-Managed Kubernetes requires the following hostname configurations:
```

Comment:
I suggest making the opening sentences to each of these three subtopics parallel: either "Before...your IT admin must" or "To function correctly...requires these configurations."

Reply:
GLOBALLY verify product name presentations.